### PR TITLE
PS-5656-doc-incorrect-encryption-5.7  

### PR DESCRIPTION
**Pull request by patrickbirch**
corrected notes and added specific version features that PXB does not support.

### CODE REVIEW

File: doc/source/management/data_at_rest_encryption.rst [hunk 1]
```diff
@@ -222,9 +222,24 @@ Trying to add unencrypted table to this table space will result in an error:
...
 .. note::
 
-   |Percona XtraBackup| currently doesn't support backup of encrypted general
+   |Percona XtraBackup| version 2.4 and version 8.0 support backup of encrypted general
    tablespaces.
 
+   |Percona XtraBackup| version 2.4 does not support the following encryption features:
+
+      * System tablespace encryption
+      * Parallel doublewrite buffer encryption
```
Comment (laurynas-biveinis) [inline doc/source/management/data_at_rest_encryption.rst]: @gl-sergei does XB care about the parallel doublewrite buffer at all?

**Review by laurynas-biveinis — COMMENTED** [review]
Need reply from @gl-sergei to finish the review

**Comment by patrickbirch** [comment]
Added Sergei - please see Laurynas comment.

**Comment by patrickbirch** [comment]
Sergei said:
Parallel doublewrite buffer encryption does not affect xtrabackup in any way. System tablespace encryption is not supported in 2.4, but will be supported in 8.0 with the next release.

**Comment by patrickbirch** [comment]
updated the doc based on Sergei's comments.

File: doc/source/management/data_at_rest_encryption.rst [hunk 2]
```diff
@@ -408,6 +415,8 @@ on every other key rotation.
 This variable simultaneously turns on the encryption of files used by InnoDB for
 full text search using parallel sorting, building indexes using merge sort, and
 online DDL logs created by InnoDB for online DDL.
+=======
+>>>>>>> Stashed changes
```
Comment (dutow) [inline doc/source/management/data_at_rest_encryption.rst]: You left a merge marker here

File: doc/source/management/data_at_rest_encryption.rst [hunk 3]
```diff
@@ -395,6 +401,7 @@ KEYRING. The value of this variable determines how frequently the encrypted
 tables should be encrypted again. If it is set to **1**, the encrypted table is
 re-encrypted on each key rotation. If it is set to **2**, the table is encrypted
 on every other key rotation.
+<<<<<<< Updated upstream
```
Comment (dutow) [inline doc/source/management/data_at_rest_encryption.rst]: You left a merge marker here

**Review by dutow — CHANGES_REQUESTED** [review]

File: doc/source/management/data_at_rest_encryption.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/management/data_at_rest_encryption.rst]: removed.

File: doc/source/management/data_at_rest_encryption.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/management/data_at_rest_encryption.rst]: removed

**Comment by dutow** [comment]
@patrickbirch now the PR contains changes for many files it shouldn't?

**Comment by patrickbirch** [comment]
Odd. OK. I'll redo this.

**Comment by dutow** [comment]
You can do

```
git reset HEAD~
git add doc/source/management/data_at_rest_encryption.rst
git commit ...
```

to recreate the commit with just that single file.

**Comment by patrickbirch** [comment]
Thanks. I'll do that.

File: doc/source/management/data_at_rest_encryption.rst [hunk 4]
```diff
@@ -222,8 +222,12 @@ Trying to add unencrypted table to this table space will result in an error:
 
 .. note::
 
-   |Percona XtraBackup| currently doesn't support backup of encrypted general
-   tablespaces.
+    |Percona XtraBackup| version 2.4 does not support the following:
```
Comment (dutow) [inline doc/source/management/data_at_rest_encryption.rst:225]: This paragraph is now indented differently than the rest of the file? Existing indentation is also inconsistent, but I don't see any other paragraphs with 4 space indents.

**Review by dutow — APPROVED** [review]
One small comment, otherwise looks good

**Comment by dutow** [comment]
Please also change the commit message as I mentioned in #3424

**Comment by patrickbirch** [comment]
replaced by 6075 - no longer needed.

**Pull request closed without merging**